src/ru/skypro/Main.java

```diff
@@ -4,5 +4,60 @@
 
     public static void main(String[] args) {
 	// write your code here
+        System.out.println("Задача 1");
+        byte child = 1;
+        short men = 200;
+        int women = 500;
+        long people = 1000L;
+        float height = 2.55f;
+        double weight = 50.7777777;
+        System.out.println("Задача 2");
+        float firstBoxer = 78.2f;
+        float secondBoxer = 82.7f;
+        float totalWeight = firstBoxer + secondBoxer;
+        System.out.println("Общий вес " + totalWeight + " кг");
+        float raznica = secondBoxer - firstBoxer;
```

Comment:
лучше стараться всегда использовать английские слова для именования переменных
```suggestion
        float delta = secondBoxer - firstBoxer;
```